libs/common/src/utils/ipfs.client.ts

```diff
@@ -98,7 +98,7 @@ export class IpfsService {
     this.logger.debug(`Hashing file buffer with length: ${fileBuffer.length}`);
     const hashed = await hasher.digest(fileBuffer);
     const hash = create(hasher.code, hashed.bytes);
-    return base58btc.encode(hash.bytes);
+    return base32.encode(hash.bytes);
```

Comment:
so this should be `base58btc` the reason is that whenever we calculate dsnp hash. the encoding should be base58btc and this is based on the question that I asked from Wil in slack.

> aramik:
> So just to be clear it looks like DSNP content hash uses blake2b-256 algorithm with base58btc encoding. is this accurate?
> Wil:
> Yes

So for CID we are going to use `base32` but for dsnp hash it is `base58btc`

Reply:
ok i will revert this one